src/core/server/http/router/request.ts

```diff
@@ -216,6 +216,13 @@ export class KibanaRequest<
   }
 
   private getEvents(request: Request): KibanaRequestEvents {
+    if (!request.raw.res) {
+      return {
+        aborted$: NEVER,
+        completed$: NEVER,
+      };
+    }
```

Comment:
What forced this change? `fromEvent` now throws when passing an `undefined` value, and wasn't previously?

Reply:
> `fromEvent` now throws when passing an `undefined` value, and wasn't previously?

That's correct. As rxjs brought better types, they also relied a bit too much on the types now. And removed many of the `typeof foo === 'undefined'` runtime validations. It essentially means that running `fromEvent(undefined, 'my-event')` throws with _Cannot read `addEventListener` of `undefined`_